[Bug] Control characters in javascript source files

It looks like some dodgy control characters have made it into the source files since `0.9.1`.

Specific pages within the application will fail when rendering affected templates with an error similar to the following:

```
Error: [$parse:lexerr] Lexer Error: Unexpected next character at columns 10-10 [Â] in expression ['DETAIL' |Â translate].
```

Using `hexdump` it is possible to see the characters in the source files:

```
$ hexdump -s 0x000002f0 -n 16 -C src/javascripts/ng-admin/Crud/show/show.html
000002f0  20 7b 7b 20 27 44 45 54  41 49 4c 27 20 7c c2 a0  | {{ 'DETAIL' |..|
00000300
```

The character sequence `0xc2 0xa0` should be `0x20`.
This occurs in several files in the project where templates have been updated for translation.

Easy fix, but logging here because the PR is going to be confusing without an explanation.

As per installation steps on page https://ng-admin-book.marmelab.com/
I  have done 
bower install ng-admin --save
and now trying html example it gives below error , is above error not fixed in latest module which bower downloads ?

angular.js:12783Error: [$parse:lexerr] Lexer Error: Unexpected next character  at columns 13-13 [Â] in expression ['NOT_FOUND' |Â translate].

--- a/src/javascripts/ng-admin/Crud/column/maJsonColumn.js
+++ b/src/javascripts/ng-admin/Crud/column/maJsonColumn.js
@@ -27,7 +27,7 @@ export default function maJsonColumn($compile) {
                 <td ng-switch="guessType(val)">
                     <ma-json-column ng-switch-when="Object" value="::val"></ma-json-column>
                     <ma-json-column ng-switch-when="Array" value="::val"></ma-json-column>
-                    <span ng-switch-when="Literal">{{ val }}</span>
+                    <span ng-switch-when="Literal">{{ val }}</span>
                 </td>
             </tr>
         </tbody>

--- a/src/javascripts/ng-admin/Crud/delete/batchDelete.html
+++ b/src/javascripts/ng-admin/Crud/delete/batchDelete.html
@@ -5,7 +5,7 @@
                 <ma-list-button ng-if="::entity.listView().enabled" entity="::entity"></ma-list-button>
             </ma-view-actions>
             <h1 compile="::batchDeleteController.title">
-                {{ 'DELETE' | translate }} {{ ::batchDeleteController.entityIds.length }} {{ ::batchDeleteController.view.entity.name() | humanize | pluralize | translate }}
+                {{ 'DELETE' | translate }} {{ ::batchDeleteController.entityIds.length }} {{ ::batchDeleteController.view.entity.name() | humanize | pluralize | translate }}
             </h1>
         </div>
     </div>

--- a/src/javascripts/ng-admin/Crud/delete/delete.html
+++ b/src/javascripts/ng-admin/Crud/delete/delete.html
@@ -5,7 +5,7 @@
                 <ma-list-button ng-if="::entity.listView().enabled" entity="::entity"></ma-list-button>
             </ma-view-actions>
             <h1 compile="::deleteController.title">
-                {{ 'DELETE' | translate }} {{ ::deleteController.view.entity.label() | humanize:true | singularize | translate }}  #{{ ::entry.identifierValue }}
+                {{ 'DELETE' | translate }} {{ ::deleteController.view.entity.label() | humanize:true | singularize | translate }}  #{{ ::entry.identifierValue }}
             </h1>
             <p class="lead" ng-if="::deleteController.description" compile="::deleteController.description">{{ ::deleteController.description }}</p>
         </div>

--- a/src/javascripts/ng-admin/Crud/form/create.html
+++ b/src/javascripts/ng-admin/Crud/form/create.html
@@ -5,7 +5,7 @@
                 <ma-list-button ng-if="::entity.listView().enabled" entity="::entity"></ma-list-button>
             </ma-view-actions>
             <h1 compile="::formController.title">
-                {{ 'CREATE_NEW' | translate }} {{ ::formController.view.entity.label() | humanize:true | singularize | translate }}
+                {{ 'CREATE_NEW' | translate }} {{ ::formController.view.entity.label() | humanize:true | singularize | translate }}
             </h1>
             <p class="lead" ng-if="::formController.description" compile="::formController.description">{{ ::formController.description }}</p>
         </div>

--- a/src/javascripts/ng-admin/Crud/show/show.html
+++ b/src/javascripts/ng-admin/Crud/show/show.html
@@ -7,7 +7,7 @@
                 <ma-delete-button ng-if="::entity.deletionView().enabled" entry="entry" entity="::entity"></ma-delete-button>
             </ma-view-actions>
             <h1 compile="::showController.title">
-                {{ ::showController.view.entity.label() | humanize | singularize | translate }}  #{{ ::entry.identifierValue }} {{ 'DETAIL' | translate }}
+                {{ ::showController.view.entity.label() | humanize | singularize | translate }}  #{{ ::entry.identifierValue }} {{ 'DETAIL' | translate }}
             </h1>
             <p class="lead" ng-if="::showController.description" compile="::showController.description">{{ ::showController.description }}</p>
         </div>

--- a/src/javascripts/ng-admin/Main/view/404.html
+++ b/src/javascripts/ng-admin/Main/view/404.html
@@ -1,7 +1,7 @@
 <div class="row">
     <div class="col-lg-12">
         <div class="page-header">
-            <h1>{{ 'NOT_FOUND' | translate }}</h1>
+            <h1>{{ 'NOT_FOUND' | translate }}</h1>
         </div>
     </div>
 </div>

